Algorythms_And_Data_Structures/Lesson_1/GeekBrains_Lesson1_Task1/GeekBrains_Lesson1_Task1/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace GeekBrains_Lesson1_Task1
+{
+    //Надеин Никита, Урок_1 по курсу Алгоритмы и Структуры данных
+    
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите целое число n: ");
+            int n = Convert.ToInt32(Console.ReadLine());
```

Comment:
Не используйте класс Convert!
```suggestion
            int n;
            Console.WriteLine("Введите целое число n");
            while(!int.TryParse(Console.ReadLine(), out n))
                Console.WriteLine("Повторите ввод n");
```